Hunk 1:
The configure script automatically detects all versions of Python and builds
the bindings for each found version. This is a problem when building outside
of a clean environment since the ports system can only build a package for one
Python flavor at a time. The found versions of Python are overridden by
environment variables set in the port Makefile.

Hunk 2:
Fix path of pkgconfig files passed to gpgrt-config.

--- configure.orig	2023-11-28 15:45:04 UTC
+++ configure
@@ -31428,6 +31428,10 @@ printf "%s\n" "$as_me: WARNING:
 
 	fi
 
+	# Override the automatically found Pythons with the selected version from ports.
+	PYTHONS=$BSD_PYTHON
+	PYTHON_VERSIONS=$BSD_PYTHON_VERSION
+	
 	# Recover some values lost in the second attempt to find Python.
 	PYTHON="$(echo $PYTHONS | cut -d ' ' -f 1)"
 	PYTHON_VERSION="$(echo $PYTHON_VERSIONS | cut -d ' ' -f 1)"
@@ -32691,7 +32695,7 @@ fi
     # Get the prefix of gpgrt-config assuming it's something like:
     #   <PREFIX>/bin/gpgrt-config
     gpgrt_prefix=${GPGRT_CONFIG%/*/*}
-    possible_libdir1=${gpgrt_prefix}/lib
+    possible_libdir1=${gpgrt_prefix}/libdata
     # Determine by using system libdir-format with CC, it's like:
     #   Normal style: /usr/lib
     #   GNU cross style: /usr/<triplet>/lib
